plugins/become/podman_unshare.py

```diff
@@ -130,8 +129,8 @@ def build_become_command(self, cmd, shell):
 
         becomecmd = 'podman unshare'
 
-        user = self.get_option('become_user') or ''
-        if user:
+        user = self.get_option('become_user') or 'root'
```

Comment:
Shouldn't it be the current user by default?

Reply:
It will run as the current user (ansible_user) if become_user is root, or not set at all (by default).
Actually, if become_user has been set to anything but root it will be executed with sudo, otherwise no sudo applied.
I tested it and was working flawlessly.

Reply:
Thanks for clarification. I think we can add some tests here.
I'll prepare a test skeleton.

Reply:
ok

Reply:
You can use this as a template for tests: https://github.com/containers/ansible-podman-collections/pull/531 Just replace `podman_lovely_module` by your module/plugin name everywhere.